services/def.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
I don't think that this file should be part of masterfiles itself. But maybe exist as example_def.json OOTB for people to get inspiration from. ?